File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/attivita/risorse/RisorsaDestinazioneUso.ts
import { DestinazioneUso } from "app/Model/metaschema/utilizzi/DestinazioneUso";
import { Risorsa } from "./Risorsa";

export class RisorsaDestinazioneUso extends Risorsa {

    destinazioneUso: DestinazioneUso;

    constructor() {
        super();
        this.destinazioneUso = new DestinazioneUso();
        this.classType = 'RisorsaDestinazioneUso';
    }

}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/attivita/risorse/RisorsaDestinazioneUso.cs
using AgronicaCoreModelsSTD.metaschema.utilizzi;

namespace AgronicaCoreModelsSTD.attivita.risorse
{
    public class RisorsaDestinazioneUso : Risorsa
    {
        public DestinazioneUso destinazioneUso { get; set; }

        public RisorsaDestinazioneUso()
        {
            classType = costanti.ClassType.RisorsaDestinazioneUso;
        }

    }
}
